root/data-emacs.yml

```diff
@@ -39,16 +39,16 @@
         help: Aller à la dernière ligne de l'historique
       - cmds:
           - Ctrl-r
-        help: Rechercher antérieurement dans l'historique
+        help: Rechercher en arrière dans l'historique
```

Comment:
« en arrière » seems strange to my North-American native French ear in this context.

What do you think of:

Rechercher **en ordre chronologique inverse** dans l'historique

I'm a humble coder though. Not a linguist.